test/e2e/runner/perturb.go

```diff
@@ -66,7 +66,7 @@ func PerturbNode(node *e2e.Node, perturbation e2e.Perturbation) (*rpctypes.Resul
 		return nil, fmt.Errorf("unexpected perturbation %q", perturbation)
 	}
 
-	status, err := waitForNode(node, 0, 10*time.Second)
+	status, err := waitForNode(node, 0, 20*time.Second)
```

Comment:
Out of curiosity, was this needed specifically for this PR or just in general?

Reply:
It's completely unrelated to this PR. I hope with this and with #295 e2e test will pass most of the times 🤞🏼